app.rb

```diff
@@ -23,6 +27,16 @@
   company_name = params['company'] || "demo-#{Time.now.to_i}"
   domain = company_name + '.com'
 
+  organizations = WorkOS::Portal.list_organizations(
```

Comment:
Moved this logic from /portal to be loaded when we load the demo. I'd prefer to have the fake demo app have an extra moment to prepare data than to make our Admin Portal itself seem slow. 

It does have the side effect of creating the demo org earlier, but I'm not super worried about that.

Soon we'll be rebuilding the demo to contain multiple demos and I'll refactor this to create the data when the demo is selected with a loading spinner if applicable.